doc/2/guides/advanced/api-keys/index.md

```diff
@@ -22,6 +22,10 @@ Administrators can create API keys for other users using the [security:createApi
 
 By default, **API keys do not expire**. But it is possible to specify the duration of an API key using the argument `expiresIn`.
 
+::: info
+It is also possible to set a maximum validity period for an API key under the key `security.apiKey.maxTTL` in the Kuzzle configuration.
```

Comment:
You should add a sentence saying that this limit only apply to API key created with the `auth` controller